config/config.go

```diff
@@ -148,6 +148,8 @@ type Config struct {
 	SkipRegisterToDashboard bool `toml:"skip-register-to-dashboard" json:"skip-register-to-dashboard"`
 	// EnableTelemetry enables the usage data report to PingCAP.
 	EnableTelemetry bool `toml:"enable-telemetry" json:"enable-telemetry"`
+	// Labels indicates the label for the tidb server
```

Comment:
please add more comments about this config. for example, why we need this config and how it's used.

Reply:
updated.